### PR TITLE
Storybook

### PR DESCRIPTION
**Pull request by keip**

### CODE REVIEW

File: src/stories/components/buttons.js [hunk 1]
```diff
@@ -1,17 +1,107 @@
...
+    padding: .5rem;
+    border: none;
+    border-radius: 5px;
+`;
+
+storiesOf('Buttons', module)
+    .add('All', () => (
+        <Wrapper>
+            <H1>
+                Default <BtnLink onClick={linkTo('Buttons', 'Button')}>{'<Button />'}</BtnLink>
```
Comment (slowbackspace) [inline src/stories/components/buttons.js:47]: I couldn't figured out that this is clickable link, can we add `cursor:pointer` on` BtnLink`?

File: src/stories/components/buttons.js [hunk 2]
```diff
@@ -1,17 +1,107 @@
...
+                </Button>
+                <Button isTransparent>
+                    Transparent
+                </Button>
+                <Button isDisabled>
+                    Disabled
+                </Button>
+            </Row>
+            <H1>
+                Inverse <BtnLink onClick={linkTo('Buttons', 'Button')}>{'<Button isInverse />'}</BtnLink>
```
Comment (slowbackspace) [inline src/stories/components/buttons.js:64]: same

File: src/stories/components/buttons.js [hunk 3]
```diff
@@ -1,17 +1,107 @@
...
+                <Button
+                    isInverse
+                    icon={icons.PLUS}
+                    isDisabled
+                >
+                    Web USB Disabled
+                </Button>
+            </Row>
+            <H1>
+                Notification <BtnLink onClick={linkTo('Buttons', 'Button Notification')}>{'<ButtonNotification />'}</BtnLink>
```
Comment (slowbackspace) [inline src/stories/components/buttons.js:87]: same

File: src/stories/components/buttons.js [hunk 4]
```diff
@@ -1,17 +1,107 @@
...
+                </ButtonNotification>
+                <ButtonNotification type="warning">
+                    Warning!
+                </ButtonNotification>
+                <ButtonNotification type="error">
+                    Error!
+                </ButtonNotification>
+            </Row>
+            <H1>
+                Pin <BtnLink onClick={linkTo('Buttons', 'Button Pin')}>{'<ButtonPin />'}</BtnLink>
```
Comment (slowbackspace) [inline src/stories/components/buttons.js:101]: last one :D

**Review by slowbackspace — APPROVED** [review]

File: src/components/Link/index.js [hunk 5]
```diff
@@ -10,7 +10,7 @@ const A = styled.a`
     text-decoration: none;
     cursor: pointer;
     transition: ${TRANSITION.HOVER};
-    font-size: ${FONT_SIZE.SMALL};
+    font-size: ${({ size = 'inherit' }) => size};
```
Comment (slowbackspace) [inline src/components/Link/index.js]: ?

Comment (slowbackspace) [reply src/components/Link/index.js]: font size should be inherit by default. I dont think we need a prop to sent font size. If anyone needs different they can still use custom css to override this default behaviour. 
eg
```
const overrideLink = styled(Link)`
  font-size: 180px;
`;
```

Comment (slowbackspace) [reply src/components/Link/index.js]: (reposting as I deleted the comment somehow)
I dont think we need a prop to set a font size. If anyone needs different one they can still use custom css to override this default behavior. 
eg with:
```
const overrideLink = styled(Link)`
  font-size: 180px;
`;
```
imho we should just remove `font-size` line

File: src/stories/components/buttons.js (hunk 1 above)
Comment (keip) [reply src/stories/components/buttons.js:47]: added cursor:pointer and hover effect

**Pull request merged**